src/Compilers/CSharp/Test/Semantic/Semantics/LambdaTests.cs

```diff
@@ -6972,10 +6972,7 @@ public static void Main(string[] args)
 ";
 
             var comp = CreateCompilation(source);
-            comp.VerifyDiagnostics(
-                // (6,9): error CS7036: There is no argument given that corresponds to the required formal parameter 'arg' of 'Func<int, int>'
-                //         lam1();
-                Diagnostic(ErrorCode.ERR_NoCorrespondingArgument, "lam1").WithArguments("arg", "System.Func<int, int>").WithLocation(6, 9));
+            comp.VerifyDiagnostics();
```

Comment:
Did we consider having two variations of `VerifyDiagnostics` here: langVersion 11 and langVersion preview?

Reply:
I was planning asking for a dedicated langversion test in the next PR, as it will be focusing on errors :).

Reply:
Are we ok to wait until a next PR for this or should I do this now?